build/dsc/templates/dsc_type_spec.rb.erb

```diff
@@ -162,6 +162,12 @@ describe Puppet::Type.type(:dsc_<%= resource.friendlyname.downcase %>) do
   end
 <%      end #  @spec_test_values['munged_ints']... -%>
 
+<%    elsif property.int_array? || property.int? || property.uint? || property.uint_array?  # if property.int? -%>
```

Comment:
They should still accept a string for Puppet 3.x.

I'll make sure I review everything carefully prior to merge.